### PR TITLE
Be more specific about about the versions affected by HTTP-01 regress…

### PR DESCRIPTION
**Pull request by wallrj**
…ion #4537

Signed-off-by: Richard Wall <richard.wall@jetstack.io>

### CODE REVIEW

**Review by jakexks — APPROVED** [review]
/lgtm

File: content/en/docs/installation/upgrading/ingress-class-compatibility.md [hunk 1]
```diff
@@ -5,7 +5,17 @@ weight: 19
...
+See [Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC].
+
+[Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC]: https://github.com/jetstack/cert-manager/issues/4537
+
+In v1.5.5, v1.6.2 and 1.7.0 we have fixed this problem.
+
+If you have cert-manager v1.5.3 (or below) you should skip v1.5.4 and upgrade to v1.5.5 then v1.6.2 and then v1.7.0
+and you can ignore the rest of this document.
+
+The following notes apply to anyone upgrading from cert-manager v1.5.4, v1.6.0, v1.6.1 on Kubernetes v1.19 or later.
```
Comment (jsoref) [inline content/en/docs/installation/upgrading/ingress-class-compatibility.md:18]: If you aren't using an Oxford comma:
```suggestion
The following notes apply to anyone upgrading from cert-manager v1.5.4, v1.6.0 or v1.6.1 on Kubernetes v1.19 or later.
```

File: content/en/docs/release-notes/release-notes-1.7.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ spec:
 
 This broke many users that either don't use an Ingress controller that supports the field (such as ingress-gce and Azure AGIC), as well as people who did not need to create an IngressClass previously (such as with Istio and Traefik).
 
-The regression is present in cert-manager 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4,
+The regression is present in cert-manager 1.5.4,
 1.6.0, 1.6.1, 1.6.2, and 1.6.3. It is only present on Kubernetes 1.19+ and only appears when using an Issuer or ClusterIssuer with an ACME HTTP-01 solver configured.
```
Comment (jsoref) [inline content/en/docs/release-notes/release-notes-1.7.md:48]: If you don't like the Oxford comma:
```suggestion
1.6.0, 1.6.1, 1.6.2 and 1.6.3. It is only present on Kubernetes 1.19+ and only appears when using an Issuer or ClusterIssuer with an ACME HTTP-01 solver configured.
```

File: content/en/docs/installation/upgrading/ingress-class-compatibility.md [hunk 3]
```diff
@@ -5,7 +5,17 @@ weight: 19
 type: "docs"
 ---
 
-The following notes apply to anyone upgrading through cert-manager 1.5 or later on Kubernetes v1.19 or later.
+In cert-manager v1.5.4 we made a change to the HTTP-01 code which was not backwards compatible.
+See [Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC].
+
+[Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC]: https://github.com/jetstack/cert-manager/issues/4537
+
+In v1.5.5, v1.6.2 and 1.7.0 we have fixed this problem.
```
Comment (jsoref) [inline content/en/docs/installation/upgrading/ingress-class-compatibility.md:13]: If you do like the Oxford comma:
```suggestion
In v1.5.5, v1.6.2, and 1.7.0 we have fixed this problem.
```

File: content/en/docs/installation/upgrading/ingress-class-compatibility.md (hunk 1 above)
Comment (jsoref) [inline content/en/docs/installation/upgrading/ingress-class-compatibility.md:18]: If you are using the Oxford comma:
```suggestion
The following notes apply to anyone upgrading from cert-manager v1.5.4, v1.6.0, or v1.6.1 on Kubernetes v1.19 or later.
```

**Comment by wallrj** [comment]
Oops I forgot to add a hold and this merged automatically.

**Pull request merged**